nala/blocks/smoke-test/smoke.page.js

```diff
@@ -15,6 +19,27 @@ export default class SmokeTest {
     this.emailPageContinueButton = page.locator('//button[@data-id="EmailPage-ContinueButton"]');
     this.passwordField = page.locator('#PasswordPage-PasswordField');
     this.passwordPageContinueButton = page.locator('//button[@data-id="PasswordPage-ContinueButton"]');
+    this.partnershipOpportunitiesSection = page.locator('#partnership-opportunities');
+    this.resellerProgram = page.locator('#reseller-program');
+    this.retailProgram = page.locator('#retail-program');
+    this.regionPicker = page.locator('.feds-regionPicker');
+    this.apacRegion = page.locator('[daa-ll="Asia Pacific English-9--Asia Pacific English"]');
+    this.salesGnavButton = page.locator('.feds-navItem:nth-of-type(2) button');
+    this.specializationButton = page.locator('.feds-menu-items ul li:nth-of-type(2) a');
+    this.vipMarketplace = page.locator('[tabindex]:has-text("VIP Marketplace")');
+    this.government = page.locator('[tabindex]:has-text("Government")');
+    this.education = page.locator('[tabindex]').filter({ hasText: /^Education$/ });
+    this.educationElite = page.locator('[tabindex]').filter({ hasText: /^Education Elite$/ });
+    this.educationEliteKorea = page.locator('[tabindex]').filter({ hasText: /^교육 기관용 Elite$/ });
+    this.worldwide = page.locator('[tabindex]:has-text("Worldwide")');
+    this.govermentElite = page.locator('[tabindex]:has-text("공공 기관용 Elite")');
+    this.adobeSubstance3D = page.locator('[tabindex]:has-text("Adobe Substance 3D")');
+    this.resellerDeProgram = page.locator('#leitfäden-zum-apc-programm');
+    this.resellerProgramLink = page.locator('h2.body-m').nth(0);
+    this.retailProgramLink = page.locator('h2.body-m').nth(1);
+    this.apacResellerProgramGuid = page.locator('[daa-ll="Reseller Program Gui-1--APC Program Guides"]');
```

Comment:
please verify the display names of these links as written in the test spec:
    Reseller Program Guide
    Distributor Guide